packages/modules/src/modules/banners/choiceCardsBanner/components/ChoiceCardFrequencyTabs.tsx

```diff
@@ -62,23 +76,25 @@ export const ChoiceCardFrequencyTabs = ({
     const getRecurringLabelText = (tabFrequency: ContributionFrequency) =>
         tabFrequency[0] + tabFrequency.slice(1).toLowerCase();
 
-    const getTabList = () => {
-        return tabFrequencies.map(tabFrequency => ({
-            frequency: tabFrequency,
-            id: `banner-${tabFrequency}`,
-            labelText: tabFrequency === 'ONE_OFF' ? 'Single' : getRecurringLabelText(tabFrequency),
-            selected: selection.frequency === tabFrequency,
-        }));
-    };
+    const tabList = tabFrequencies.map(tabFrequency => ({
+        frequency: tabFrequency,
+        id: `banner-${tabFrequency}`,
+        labelText: tabFrequency === 'ONE_OFF' ? 'Single' : getRecurringLabelText(tabFrequency),
+        selected: selection.frequency === tabFrequency,
+    }));
 
     return (
-        <Box>
-            <BannerChoiceCardsPaymentFrequencyTabs
-                ariaLabel="payment frequency tabs"
-                tabs={getTabList()}
-                selectedTab={selection.frequency}
-                onTabChange={updateFrequency}
-            />
-        </Box>
+        <div css={container}>
+            {tabList.map(tab => (
+                <ChoiceCard
```

Comment:
Can see simplified and removed custom banner choice cards to use choice cards, very neat and tidy!